x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/list_page/index.tsx

```diff
@@ -70,61 +162,116 @@ export const AgentConfigListPage: React.FunctionComponent<{}> = () => {
+    EuiTableFieldDataColumnType<AgentConfig> | EuiTableActionsColumnType<AgentConfig>
+  > => {
+    const cols = [
+      {
+        field: 'name',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.nameColumnTitle', {
+          defaultMessage: 'Name',
+        }),
+        // FIXME: use version once available - see: https://github.com/elastic/kibana/issues/56750
+        render: (name: string, agentConfig: AgentConfig) => (
+          <EuiFlexGroup gutterSize="s" wrap={true} alignItems="baseline">
+            <EuiFlexItem grow={false}>
+              <EuiLink href={`${DETAILS_URI}${agentConfig.id}`}>{name || agentConfig.id}</EuiLink>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText color="subdued" size="xs">
                 <FormattedMessage
-                  id="xpack.ingestManager.agentConfigList.viewActionLinkText"
-                  defaultMessage="view"
+                  id="xpack.ingestManager.agentConfigList.revisionNumber"
+                  defaultMessage="rev. {revNumber}"
+                  values={{ revNumber: '999' }}
                 />
-              </EuiLink>
-            );
-          },
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        ),
+      },
+      {
+        field: 'description',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.descriptionColumnTitle', {
+          defaultMessage: 'Description',
+        }),
+        render: (description: AgentConfig['description']) => (
+          <EuiTextColor color="subdued">{description}</EuiTextColor>
+        ),
+        width: '30%',
+      },
+      {
+        field: 'updated_on',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.updatedOnColumnTitle', {
+          defaultMessage: 'Last updated on',
+        }),
+        render: (date: AgentConfig['updated_on']) => (
+          <FormattedDate value={date} year="numeric" month="long" day="2-digit" />
+        ),
+      },
+      {
+        field: 'agents',
+        name: i18n.translate('xpack.ingestManager.agentConfigList.agentsColumnTitle', {
+          defaultMessage: 'Agents',
+        }),
+        render: (agents: unknown, config: AgentConfig) => {
+          // FIXME: implement agents once known in API/Schema
+          const agentCount = 99;
+          const displayValue = (
+            <FormattedMessage
+              id="xpack.ingestManager.agentConfigList.agentsText"
+              defaultMessage="{agentCount, plural, one {# agent} other {# agents}}"
+              values={{ agentCount }}
+            />
+          );
+          return agentCount > 0 ? (
+            <EuiLink href={`${FLEET_URI}?kuery=agents.config_id : ${config.id}`}>
+              {displayValue}
+            </EuiLink>
+          ) : (
+            displayValue
+          );
         },
-      ],
-      width: '100px',
-    },
-  ];
+      },
+      {
+        field: 'datasources',
```

Comment:
add dataType: 'number' to right align the "datasources" column

Reply:
done